dotcom-rendering/src/web/components/CommercialMetrics.importable.tsx

```diff
@@ -30,6 +31,7 @@ export const CommercialMetrics = ({ enabled }: Props) => {
 		const clientSideTestsToForceMetrics: ABTest[] = [
 			/* keep array multi-line */
 			integrateIma,
+			noCarrotAdsNearNewsletterSignupBlocks,
```

Comment:
Does this also need adding to `CoreVitals.importable` ?

Or there's another way to do this from the test directly as in the `liveblog-desktop-outstream` test:

```
	variants: [
		{ id: 'control', test: () => bypassMetricsSampling },
		{ id: 'variant', test: () => bypassMetricsSampling },
	],
```

Reply:
Thanks, I've amended it to use the new approach